server/jobHandler/src/index.ts

```diff
@@ -52,6 +54,12 @@ const jobHandler = async (context: AWSLambda.Context, key: string): Promise<{ is
 	}
 	//Set the job status to processing, 
 	await updateJobStatus(key, JobStatus.PROCESSING)
+	// send email telling user job is processing
+	try {
+		await sendProcessingEmail(key);
```

Comment:
This email should be sent before we update the jobStatus, as the jobHandler will call itself meaning that multiple emails would be sent. Currently the jobStatus isn't being set by the uploadHandler, but as we know the jobStatus isn't generating, it's safe to just check if the jobStatus isn't processing.
i.e.
```
if(job.jobStatus != JobStatus.PROCESSING){
    // send email telling user job is processing
    try {
	await sendProcessingEmail(key);
    } catch (error) {
	throw Error(`failed to send processing email for job:${job.id}`);
    }
    
    //Set the job status to processing, 
    await updateJobStatus(key, JobStatus.PROCESSING)
}
```

Reply:
Ah okay didn't realise it was recursive - should be fixed now